src/main/scala/kanaloa/reactive/dispatcher/metrics/MetricsCollector.scala

```diff
@@ -1,8 +1,9 @@
-package com.iheart.workpipeline.metrics
+package kanaloa.reactive.dispatcher.metrics
 
 import akka.actor.ActorSystem
 import com.typesafe.config.{ Config, ConfigFactory }
-import com.iheart.util.ConfigWrapper.ImplicitConfigWrapper
+import kanaloa.util.ConfigWrapper
+import ConfigWrapper.ImplicitConfigWrapper
```

Comment:
`ConfigWrapper` isn't used directly anywhere, probably just change this to `import kanaloa.util.ConfigWrapper.ImplicitConfigWrapper`.

Also in general I feel like imports should be fully qualified package names, so there's no ambiguity as to which package it came from (especially in the other imports like `import Queue.something`)